services/fxa/serverless.yml

```diff
@@ -142,6 +48,7 @@ custom:
     stage: restricted
     qa: restricted
     dev: unfettered
+    fab: unfettered
```

Comment:
Do we still need the fab environment as part of landing this or was that just an ad hoc environment to get around the too many chefs in the kitchen issue in the development environment.

Reply:
i created a new env because there has seemed to be pressure to do so; once created i am ok leaving it for now